src/main/java/com/transferwise/banks/demo/utilities/web/DemoUtilitiesController.java

```diff
@@ -0,0 +1,45 @@
+package com.transferwise.banks.demo.utilities.web;
+
+import com.transferwise.banks.demo.customer.domain.Customer;
+import com.transferwise.banks.demo.customer.domain.CustomersService;
+import com.transferwise.banks.demo.customer.domain.address.Address;
+import com.transferwise.banks.demo.customer.domain.occupation.Occupation;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Offers endpoints specifically for demo purposes on the Mobile Reference App.
+ *
+ * ⚠️ This is only part of the repository for demo purposes, but has no value as reference code. ⚠️
+ */
+
+@RestController
+@RequestMapping("demo")
+public class DemoUtilitiesController {
+
+    private final CustomersService customersService;
+    private final DemoUtilities demoUtilitiesService;
+
+    public DemoUtilitiesController(CustomersService customersService) {
+        this.customersService = customersService;
+        this.demoUtilitiesService = new DemoUtilities();
+    }
+
+    @PostMapping("new-customer")
+    public Long createNewDemoCustomer() {
+        List<Occupation> occupations = new ArrayList<>();
+        occupations.add(new Occupation("Clown", "FREE_FORM"));
+        Address address = new Address("56, Shoreditch High Street", "EC1v 6JJ", "London", "", "GB", occupations);
```

Comment:
I think that's not a valid postcode, would that cause any issues in the backend?
There's gonna be a lot of clowns in shoreditch

Reply:
This should be fine. It works fine when I run it locally so should not cause an issue.

Reply:
Since I am making changes I can update it.

Reply:
Watch out there's a test using that wrong postcode that will break

Reply:
That is not in my code either.